apps/ios/Shared/Views/Chat/ComposeMessage/ComposeView.swift

```diff
@@ -167,6 +167,13 @@ struct ComposeState {
         }
     }
 
+    var manyMediaPreviews: Bool {
```

Comment:
limited live message to a single attachment but there's still a bug when starting live message with a single attachment that each new live message send is a separate send